src/index.ts

```diff
@@ -29,6 +38,26 @@ import type {
  * @returns account
  */
 async function createRhinestoneAccount(config: RhinestoneAccountConfig) {
+  function deploy(chain: Chain, session?: Session) {
+    return deployInternal(config, chain, session)
+  }
+
+  function prepareTransaction(transaction: Transaction) {
+    return prepareTransactionInternal(config, transaction)
+  }
+
+  function signTransaction(transaction: Transaction, bundleData: BundleData) {
+    return signTransactionInternal(config, transaction, bundleData)
+  }
+
+  function submitTransaction(
+    transaction: Transaction,
+    bundleData: BundleData,
+    signature: Hex,
+  ) {
+    return submitTransactionInternal(config, transaction, bundleData, signature)
+  }
```

Comment:
Leaning towards a separate function (vs reusing `sendTransaction`) to separate those concepts (high-level vs low-level API). I think eventually we will deprecate the high function and rename `submitTransaction` → `sendTransaction`

Reply:
yeah I'm a fan of one func but if u think this is better for now then lets do it